bin/cron/schedule.js

```diff
@@ -1,20 +1,25 @@
-var Cli = require('structured-cli');
-var Logs = require('../../lib/logs');
-var PrintCronJob = require('../../lib/printCronJob');
-var ValidateCreateArgs = require('../../lib/validateCreateArgs');
-var WebtaskCreator = require('../../lib/webtaskCreator');
-var _ = require('lodash');
+'use strict';
 
+const Cli = require('structured-cli');
+const Logs = require('../../lib/logs');
+const PrintCronJob = require('../../lib/printCronJob');
+const ValidateCreateArgs = require('../../lib/validateCreateArgs');
+const WebtaskCreator = require('../../lib/webtaskCreator');
+const _ = require('lodash');
 
-var intervals = {
+const CRON_AUTH_MIDDLEWARE = '@webtask/cron-auth-middleware';
```

Comment:
Wasn't this supposed to live in webtask-userspace?

Reply:
This _is_ `webtask-userspace`: https://github.com/auth0/webtask-userspace/blob/master/packages/middleware-compiler/package.json#L2